### PR TITLE
implements answer key styling for accounting

### PR DESCRIPTION
**Pull request by kswanson33**
#2895 

image
image

### CODE REVIEW

**Review by LostMojo — APPROVED** [review]
Looks good!

File: styles/designs/cardboard/parts/_answerkey-shapes.scss [hunk 1]
```diff
@@ -1,12 +1,14 @@
 @import 'answerkey-components';
 @import 'exercises-components';
+@import 'titles-components';
```
Comment (brittany-johnson) [inline styles/designs/cardboard/parts/_answerkey-shapes.scss:3]: Is this import needed? I noticed that the component you added is from `answerkey-components` so just want to make sure.

**Review by brittany-johnson — APPROVED** [review]
Just left one comment - other than that looks good!

File: styles/designs/cardboard/parts/_answerkey-shapes.scss (hunk 1 above)
Comment (kswanson33) [reply styles/designs/cardboard/parts/_answerkey-shapes.scss:3]: You're right! I'll take it out. Thanks!

**Pull request merged**